mocha/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "mocha",
+  "version": "1.0.0",
+  "description": "Mocha API integration tests",
+  "scripts": {
+    "test": "mocha './mocha/test/**/*.js' --recursive --timeout 60000 --exit"
```

Comment:
I think the test path should be changed to the following instead: 
"test": "mocha './test/**/*.js' --recursive --timeout 60000 --exit"

I ran the test on my local machine while in the parent folder using the original test path provided and it triggered an error because no package.json file exists in the parent folder. I ran the test while in the mocha folder using what I suggested above and it worked. 

![image](https://user-images.githubusercontent.com/56090896/90303516-89f3f900-de7c-11ea-9d7e-534b63314749.png)

Reply:
@jessica-lau 
How do you want to change the test path?

Reply:
@galeza currently the test path in the package.json is this:
"test": "mocha './mocha/test/**/*.js' --recursive --timeout 60000 --exit"

but it should be changed to this:
"test": "mocha './test/**/*.js' --recursive --timeout 60000 --exit"

Reply:
@jessica-lau 

I added devDependencies to package.json and modified also docker-compose to start tests using :_'./test/**/*.js'_
Please verify.